spring-data-rest/src/main/java/com/baeldung/config/RestConfig.java

```diff
@@ -3,12 +3,14 @@
 import com.baeldung.models.WebsiteUser;
 import com.baeldung.projections.CustomBook;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 import org.springframework.data.rest.core.config.RepositoryRestConfiguration;
 import org.springframework.data.rest.core.mapping.ExposureConfiguration;
 import org.springframework.data.rest.webmvc.config.RepositoryRestConfigurer;
 import org.springframework.http.HttpMethod;
 
 @Configuration
+@Profile("!expose-ids")
```

Comment:
It might be a good idea to put a comment next to this to explain why you've done it.